### PR TITLE
Mysupply search backed by candidate search

### PR DESCRIPTION
**Pull request by bobthearsonist**
This simply captures a quick how-to guide on searching mycandidates data. It can be expanded and improved as we field more request and get more new customers.

Its intent is to replace some of the contextual knowledge in the Search API documentation we had(have) for the mysupply search endpoint.

My next pull request will add all of the field descriptions form that document here as it is valuable information even after the endpoint dies.

### CODE REVIEW

**Comment by bobthearsonist** [comment]
@thomasmahoney how does this look for a first stab?

File: SearchingPrivateCandidates.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+In order to search you will need to know a little about the customer. In mysupply the customer matches the clientkey. A clientkey is a mysupply wrapper for an accountdid. To get the clientkey for an accountdid [you will want to use the Client API](https://github.com/careerbuilder/CandidateDataServicesDocumentation/blob/master/ClientAPI.md#get-1)
```
Comment (andhesmi) [inline SearchingPrivateCandidates.md:2]: casing here?

File: SearchingPrivateCandidates.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ 
+To see the makeup of the customer do a [GET on the customer API](https://github.com/careerbuilder/CandidateDataServicesDocumentation/blob/master/CustomerAPI.md) where customer=clientkey.
```
Comment (andhesmi) [inline SearchingPrivateCandidates.md:4]: "to see information about a customer"

File: SearchingPrivateCandidates.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+ 
+With the customer information you then construct a [search request](http://careerbuildersearch.com/docs/candidate). The key is to limit the values and facets you request to only those that you need. This example demonstrates what a more complex search looks like.
```
Comment (andhesmi) [inline SearchingPrivateCandidates.md:6]: beef comment - I wouldn't say the key is to limit the values and facets you need, but we recommend that.

File: README.md [hunk 4]
```diff
@@ -2,14 +2,20 @@
 
 CDS APIs are concentrated into two areas: Candidate Data & Candidate Search.
 
-## Candidate Data APIs
+## Candidate Data
+### Uses
+* [Searching Private Candidates](/SearchingPrivateCandidates.md)
```
Comment (andhesmi) [inline README.md:7]: I would move this under Candidate Search instead

**Review by andhesmi — APPROVED** [review]
a few comments but otherwise looks good

File: README.md (hunk 4 above)
Comment (bobthearsonist) [reply README.md:7]: I left it there because its specific to using candidate search for Candidate Data aka mycandidates

**Pull request merged**